### PR TITLE
More australium-able items

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

Surgery tools, kitchen tools, nullrods and just tools in general are now australium-able

## Why It's Good For The Game

Why not

## Changelog
:cl:
add: More australium-able items
/:cl:

### CODE REVIEW

**Comment by thelaughingbomb** [comment]
Nice

**Review by thelaughingbomb — APPROVED** [review]

**Comment by TheFakeElon** [comment]
Fix bee DME includes please

**Comment by TheFakeElon** [comment]
Why not just make it some you can use Australium on any item tbh

**Comment by MCterra10** [comment]
Australium toolbox or U gey

File: austation/code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	description = "A mysterious metal element that can adapt and transform itself into different states and forms, can make subjects appear down-under."
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 	taste_description = "australia"
+	var/static/list/whitelist = typesof(/obj/item/gun) + typesof(/obj/item/melee) + typesof(/obj/item/kitchen) + typesof(/obj/item/crowbar) + typesof(/obj/item/screwdriver) + typesof(/obj/item/wrench) + typesof(/obj/item/wirecutters) + typesof(/obj/item/weldingtool) + typesof(/obj/item/retractor) + typesof(/obj/item/retractor) + typesof(/obj/item/hemostat) + typesof(/obj/item/cautery) + typesof(/obj/item/surgicaldrill) + typesof(/obj/item/scalpel) + typesof(/obj/item/circular_saw) + typesof(/obj/item/nullrod) + typesof(/obj/item/storage/toolbox)
```
Comment (Dingo-Dongler) [inline austation/code/modules/reagents/chemistry/reagents/other_reagents.dm:7]:
```suggestion
	var/static/list/whitelist = typesof(/obj/item/gun) + typesof(/obj/item/melee) + typesof(/obj/item/kitchen) + typesof(/obj/item/crowbar) + typesof(/obj/item/screwdriver) + typesof(/obj/item/wrench) + typesof(/obj/item/wirecutters) + typesof(/obj/item/weldingtool) + typesof(/obj/item/retractor) + typesof(/obj/item/hemostat) + typesof(/obj/item/cautery) + typesof(/obj/item/surgicaldrill) + typesof(/obj/item/scalpel) + typesof(/obj/item/circular_saw) + typesof(/obj/item/nullrod) + typesof(/obj/item/storage/toolbox)
```

Comment (Dingo-Dongler) [reply austation/code/modules/reagents/chemistry/reagents/other_reagents.dm:7]: Removes the duplicate retractor

**Pull request merged**